<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>demo1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_work">
        <property name="text">
         <string>当前工作项：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_status">
        <property name="text">
         <string>工作状态：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pgb">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_chooseFile">
        <property name="text">
         <string>选择源文件：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_input">
          <property name="text">
           <string>输入路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_input">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tb_input">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../img/file.png</normaloff>../img/file.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_output">
          <property name="text">
           <string>输出路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_output">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tb_output">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../img/file.png</normaloff>../img/file.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_start_pick">
          <property name="text">
           <string>开始提取实验报告</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_prj_path">
          <property name="text">
           <string>代码检测项目路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_prj_path">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tb_prj_path">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../img/file.png</normaloff>../img/file.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_type">
          <property name="text">
           <string>项目类型：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_java">
          <property name="text">
           <string>Java</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_cpp">
          <property name="text">
           <string>C++</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_py">
          <property name="text">
           <string>python</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_scan">
          <property name="text">
           <string>开始检测代码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_get">
          <property name="text">
           <string>获取检测结果</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_problem">
        <property name="text">
         <string>代码问题：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>问题类型</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>所在源文件</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>行数</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>严重性</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>问题描述</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_dc_path">
          <property name="text">
           <string>相似度检测文件路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_dc_path">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tb_dc_path">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_algrthm">
        <property name="text">
         <string>算法选择：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_a">
        <property name="text">
         <string>算法A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_b">
        <property name="text">
         <string>算法B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_c">
        <property name="text">
         <string>算法C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_d">
        <property name="text">
         <string>算法D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_dc_start">
        <property name="text">
         <string>开始检测相似度</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_2">
      <column>
       <property name="text">
        <string>文件名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>算法A</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>相似度A</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>算法B</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>相似度B</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>算法C</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>相似度C</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>算法D</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>相似度D</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
